<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopBar</class>
 <widget class="QWidget" name="TopBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255)
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(18, 183, 245);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>QQ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="maxBtn">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>2</y>
     <width>47</width>
     <height>32</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#maxBtn{border:none;}
QPushButton#maxBtn:pressed{
}
QPushButton#maxBtn:hover{
background-color: rgb(169, 169, 169);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/widgets/max.png</normaloff>:/images/widgets/max.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>552</x>
     <y>2</y>
     <width>47</width>
     <height>32</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#closeBtn{border:none;}
QPushButton#closeBtn:pressed{
}
QPushButton#closeBtn:hover{
background-color: rgb(255, 0, 0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/widgets/close.png</normaloff>:/images/widgets/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>32</height>
    </size>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="minBtn">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>2</y>
     <width>47</width>
     <height>32</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#minBtn{border:none;}
QPushButton#minBtn:pressed{
}
QPushButton#minBtn:hover{
background-color: rgb(169, 169, 169);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/widgets/min.png</normaloff>:/images/widgets/min.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>32</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
